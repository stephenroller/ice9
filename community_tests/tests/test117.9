# break in else clause										succeed

fa i := 0 to 5 ->
	if i = 0 ->
		write 0;
	[] i = 1 ->
		write 1;
	[] i = 2 ->
		write 2;
	[] i = 3 ->
		write 3;
	[] else ->
		write 4;
		break;
		write 5;
	fi
af

write 6;
