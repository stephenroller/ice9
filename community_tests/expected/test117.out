Loading file: generated/test117.tm
0 
1 
2 
3 
4 
6 
Number of instructions executed = 307
